datamodel/datamodel.go

```diff
@@ -38,6 +38,8 @@ type ModelTopicConfig struct {
 	Key string
 	// Timestamp is the timestamp field for the topic
 	Timestamp string
+	//TTL is the duration the message is valid
+	TTL time.Duration
```

Comment:
small nit. add space between `//` and `TTL`

Reply:
👍